micro-ui/web/micro-ui-internals/packages/react-components/src/hoc/ViewComposer/index.js

```
@@ -196,13 +196,19 @@ const ViewComposer = ({ isLoading = false, data, cardErrors, ...props }) => {
           ?.filter((card) => card?.navigationKey)
           ?.map((card, cardIdx) => {
-            const { sections } = card;
+            const { sections, noCardStyle } = card;
             return (
-              <Card style={activeNav && card.navigationKey ? (activeNav !== card.navigationKey ? { display: "none" } : {}) : {}} className={`employeeCard-override ${card?.cardStyle ? card?.cardStyle : ""}`}>
-                {sections?.map((section, sectionIdx) => {
-                  return renderCardSectionJSX(section);
-                })}
-              </Card>
+              noCardStyle ?
+                <div>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)